optimum/habana/transformers/modeling_utils.py

```diff
@@ -363,3 +367,9 @@ def adapt_transformers_to_gaudi():
     transformers.models.speecht5.modeling_speecht5.SpeechT5SpeechDecoderPrenet.forward = (
         gaudi_SpeechT5SpeechDecoderPrenet_forward
     )
+
+    # Optimization for stablelm on Gaudi
```

Comment:
make the indentation same

Reply:
I think the indentation of 4 spaces is already the same?